src/guide/views/dropdown/DropdownExample.jsx

```diff
@@ -67,23 +68,62 @@ export default class DropdownExample extends Component {
 
         <Example>
           <Dropdown
+            labelProvider={this.dropdownLabelProvider}
+            onSelect={this.onSelectDropdownOption}
             options={this.dropdownItems}
+            optionLabelProvider={this.dropdownOptionLabelProvider}
             selectedOption={this.state.selectedDropdownOption}
-            onSelect={this.onSelectDropdownOption}
+          />
+        </Example>
+
+        <Example title="With readonly">
+          <Dropdown
+            isReadonly
             labelProvider={this.dropdownLabelProvider}
             optionLabelProvider={this.dropdownOptionLabelProvider}
+            onSelect={this.onSelectDropdownOption}
           />
         </Example>
 
-        <Example title="With readonly">
+        <Example title="Borderless">
           <Dropdown
+            isBorderless
             labelProvider={this.dropdownLabelProvider}
+            onSelect={this.onSelectDropdownOption}
+            options={this.dropdownItems}
             optionLabelProvider={this.dropdownOptionLabelProvider}
+            selectedOption={this.state.selectedDropdownOption}
+          />
+        </Example>
+
+        <Example title="Borderless with dot">
+          <Dropdown
+            dotColor={DropdownDot.COLOR.BLUE}
```

Comment:
Yes, in regards of the mockup, remember the line item dropdowns for the chart? There I assume that a newly selected line item doesn't change the color for the chart. That among other reasons is why the `StatusDropdown` is still a component for itself.